src/pages/dashboard/WorstLinesChart/LineHbarChart/HbarChart/HbarChart.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { HbarChart, Entry } from './HbarChart'
+
+const meta = {
+  title: 'Components/MapLayers/HbarChart',
+  component: HbarChart,
+  tags: ['map', 'tooltip', 'autodocs'],
+} satisfies Meta<typeof HbarChart>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+const operatorsPT: Entry[] = [
```

Comment:
since stories are just demos, the array can be shorter (you don't have to use real data)